sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -337,7 +345,9 @@ protected ServiceBusProcessor()
 
                 if (_processErrorAsync != value)
                 {
+#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
```

Comment:
Can I ask why are we adding these warnings?

Reply:
I enabled FxCop so I had to override these errors.